src/PythonMigrationViewExtension/Differ/RichTextBoxHandler.cs

```diff
@@ -57,6 +57,8 @@ private static void ShowLineDiffs(RichTextBox diffBox, DiffPaneModel model, stri
                     case ChangeType.Deleted:
                         paragraphs.Add(ShowSubPieceDiffs(line, true));
                         break;
+                    case ChangeType.Imaginary:
+                        break;
```

Comment:
@mjkkirschner can we add a test for this?

Reply:
yes, I've already merged this, but I should be able to add a test in a followup pr.